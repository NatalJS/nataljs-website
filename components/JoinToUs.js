import React from 'react';

const JoinToUs = () => (
  <section id="join-us" className="join-us">
    <h1 className="section-title">Faça Parte</h1>
    <ul className="join-us__list">
      <li className="join-us__item">Vá aos encontros!</li>
      <li className="join-us__item">Use a hasttag #nataljs</li>
      <li className="join-us__item">Divulgue nas redes socias</li>
      <li className="join-us__item">
        Contribua em
        {' '}
        <a className="join-us__link" href="https://github.com/NatalJS" title="NatalJS" target="_blank" rel="noopener noreferrer">
          nossos projetos
        </a>
      </li>
    </ul>
  </section>
);

export default JoinToUs;
