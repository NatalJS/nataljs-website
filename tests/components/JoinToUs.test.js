import renderer from 'react-test-renderer';
import JoinToUs from 'components/JoinToUs';

describe('Component JoinToUs', () => {
  let Component;

  beforeAll(() => {
    Component = setup(JoinToUs);
  });

  it('should render component correctly', () => {
    const wrapper = renderer.create(Component());

    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
