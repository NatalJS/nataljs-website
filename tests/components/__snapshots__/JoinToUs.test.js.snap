// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component JoinToUs should render component correctly 1`] = `
<section
  className="join-us"
  id="join-us"
>
  <h1
    className="section-title"
  >
    Faça Parte
  </h1>
  <ul
    className="join-us__list"
  >
    <li
      className="join-us__item"
    >
      Vá aos encontros!
    </li>
    <li
      className="join-us__item"
    >
      Use a hasttag #nataljs
    </li>
    <li
      className="join-us__item"
    >
      Divulgue nas redes socias
    </li>
    <li
      className="join-us__item"
    >
      Contribua em
       
      <a
        className="join-us__link"
        href="https://github.com/NatalJS"
        rel="noopener noreferrer"
        target="_blank"
        title="NatalJS"
      >
        nossos projetos
      </a>
    </li>
  </ul>
</section>
`;
